.ci/install_clear_image.sh

```diff
@@ -0,0 +1,50 @@
+
+#!/bin/bash
+#
+# Copyright (c) 2017 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -e
+
+if [ "$#" -ne 2 ]; then
+    echo "Usage: $0 CLEAR_RELEASE PATH"
```

Comment:
Well, we now have 3 scripts that all hard-code the download URL. I'd still argue that `install_cc_env_ubuntu.sh` doesn't need to know that URL since it's querying for `latest` anyway.

Then, to avoid the last bit of duplication `install_clear_image.sh` and `install_clear_kernel.sh`  could both source a script/config file for that URL. I know the URL is unlikely to change, but that isn't the point here.

https://github.com/01org/cc-oci-runtime/tree/master/.ci/ gets all this right and has been in use for a long time. But we now have these scripts plus the bunch in https://github.com/clearcontainers/runtime/tree/master/.ci all doing it slightly differently. It's getting rather confusing. Is the plan to migrate all the scripts from the `runtime` to the `tests` repo?